source/documentation/doc-Introduction_to_the_VM_Portal/topics/Creating_virtual_machines.adoc

```diff
@@ -4,24 +4,35 @@
 
 *Creating a virtual machine*
 
-. Click the btn:[Create Virtual Machine] button in the toolbar.
+. Click btn:[Create Virtual Machine] in the toolbar.
+. Fill in the required information in the Create Virtual Machine wizard.
+There are four screens to complete:
```

Comment:
I strongly recommend that you not mention screen titles or individual fields unless necessary because that will make the documentation harder to maintain. You only need to mention fields that are either not self-explanatory or fields that have limitations (VM name).

Reply:
You missed mentioning few fields:
- Start virtual machine on creation
- Enable Cloud-init/Sysprep,  
- Cloud-init/SysPrep Options ()

most of those fields have the same limitation/validations as for webadmin